scaladoc-js/src/searchbar/SearchbarComponent.scala

```diff
@@ -132,7 +133,19 @@ class SearchbarComponent(engine: SearchbarEngine, inkuireEngine: InkuireJSSearch
           loading.appendChild(animation)
           properResultsDiv.appendChild(loading)
           inkuireEngine.query(query) { (m: InkuireMatch) =>
-            properResultsDiv.appendChild(m.toHTML)
+            var next: Option[Element] = None
+            0.until(properResultsDiv.children.length).foreach { i =>
+              val child = properResultsDiv.children(i)
+              val attr = child.getAttribute("mq")
+              if attr != null && attr != "" && Integer.parseInt(attr) > m.mq && next.isEmpty then {
+                next = Some(child)
+              }
+            }
```

Comment:
You should be able to do ([See this](https://github.com/scala-js/scala-js-dom/issues/194)):
```
properResultsDic.children.foreach { child =>

}
```
and also you can check if attr exists with `child.hasAttribute("mq")`.
Couldn't you make there some `foldLeft` instead of var with foreach?

Reply:
Ooooh, thanks this magic import is really helpful :smile: